MVC/controllers/interestsUsersController.js

```diff
@@ -0,0 +1,31 @@
+const { selectIntrestsByUserId, postUserInterest, deleteInterest } = require("../models/interestsUsersModel");
+
+exports.getUserInterests = (req, res) => {
+    selectIntrestsByUserId(req.params.id)
```

Comment:
Misspelling of interests, may cause problems in future

Reply:
Thank you :)